packages/maskbook/src/plugins/RedPacket/UI/RedPacket.tsx

```diff
@@ -179,6 +195,21 @@ export function RedPacket(props: RedPacketProps) {
         else if (canRefund) await refundCallback()
     }, [canClaim, canRefund, claimCallback, refundCallback])
 
+    if (isMetamaskLocked)
+        return (
+            <Card
+                className={classNames(classes.root, {
+                    [classes.metamaskContent]: true,
+                    [classes.cursor]: true,
+                })}
+                onClick={() => Services.Ethereum.popupMetaMaskUnlocked()}
```

Comment:
`onClick={Services.Ethereum.popupMetaMaskUnlocked}` does not work

Reply:
👀 interesting